program/tests/tests/solana_assumptions.rs

```diff
@@ -110,7 +110,8 @@ async fn measure_staking_rewards(mode: StakeMode) -> Lamports {
     // Move ahead one epoch so the stake becomes active.
     context.advance_to_normal_epoch(1);
 
-    let balance_t0 = context.get_sol_balance(vote_account).await;
+    let mut balance_t0 = context.get_sol_balance(vote_account).await;
+    balance_t0 = (balance_t0 + context.get_sol_balance(stake_account).await).unwrap();
```

Comment:
No need for this to be mutable, we can define it as a sum right away.

Reply:
Yes, but it looks ugly IMHO

```rust
    let balance_t0 = (context.get_sol_balance(vote_account).await
        + context.get_sol_balance(stake_account).await)
        .unwrap();
```

Anyway, I fixed it.